internal/format/markdown_document.go

```diff
@@ -17,6 +17,19 @@ const (
 		{{- end -}}
 	{{ end -}}
 	`
+	documentResourcesTpl = `
+	{{- if .Settings.ShowResources -}}
+		{{ indent 0 "#" }} Resources
+		{{ if not .Module.Resources }}
+			No resources.
+		{{ else }}
+			The following resources are used by this module:
+			{{- range .Module.Resources }}
```

Comment:
Nit: One empty line before this is missing, which causes markdown linter to complain. Maybe just remove `-`?

```
{{ range .Module.Resource }}
  ...
```